src/test/java/HomePageTest.java

```diff
@@ -15,6 +23,65 @@ public static void main(String[] arg){
         String title = driver.getTitle();
         Assert.assertEquals(title, "Kwidos");
         driver.quit();
+    }
+
+    @Test
+    public void Homework1() throws InterruptedException {
+        // Go to the homepage https://tstkwidos.tk/
+        // Verify title is equal to kwidos
+        // turn the log off
+        System.setProperty("webdriver.chrome.silentOutput","true");
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://testkwidos.tk/auth/register/contractor");
+        driver.manage().window().maximize();
```

Comment:
Please move driver configuration to another method, because you use this configuration in several test methods and also test should follow this AAA pattern: https://cypriops.com/blog/test-food-chain/